src/qt/sendcoinsdialog.cpp

```diff
@@ -652,6 +652,16 @@ void SendCoinsDialog::updateFeeMinimizedLabel()
     }
 }
 
+CoinControlDialog* SendCoinsDialog::coinControlDialog()
+{
+    if (!m_coin_control_dialog) {
+        m_coin_control_dialog = new CoinControlDialog(platformStyle, this);
```

Comment:
Assigning a parent to a `QDialog` widget changes its default location.
Is it desirable?

Reply:
I think so, at least it looked fine to me.